agent/src/com/cloud/agent/Agent.java

```diff
@@ -464,7 +493,7 @@ protected void processRequest(final Request request, final Link link) {
 
             for (int i = 0; i < cmds.length; i++) {
                 final Command cmd = cmds[i];
-                Answer answer;
+                Answer answer = null;
```

Comment:
this can be removed, an uninitialized variable is `null`, it just states it explicitly.